scripts/startUserMigration.js

```diff
@@ -0,0 +1,60 @@
+const { program } = require('commander');
+const appPromise = require('../src/app');
+const { alert, error } = require('../src/logger');
+const { schoolModel } = require('../src/services/school/model');
+const systemModel = require('../src/services/system/model');
+
+program
+	.requiredOption('--systemId <value>', 'ID of the system, which will be added to the schools')
+	.option(
+		'--officialSchoolNumbers <value...>',
+		'Official school numbers of the schools that should be migrated (space seperated values). If not given, all schools having an official school number will be migrated.'
+	);
+
```

Comment:
```suggestion
program.outputHelp();
```

Reply:
This way, the help will always be printed if we run the script. Is this, what we want to do here? Otherwise the help will be displayed if the script gets called with `-h` or `--help`.